cyral/internal/tokensettings/schema_loader.go

```diff
@@ -0,0 +1,91 @@
+package tokensettings
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/cyralinc/terraform-provider-cyral/cyral/core"
+	"github.com/cyralinc/terraform-provider-cyral/cyral/utils"
+)
+
+const (
+	MaxValidityKey              = "max_validity"
+	DefaultValidityKey          = "default_validity"
+	MaxNumberOfTokensPerUserKey = "max_number_of_tokens_per_user"
+	OfflineTokenValidationKey   = "offline_token_validation"
+)
+
+func getAccessTokenSettingsSchema(isDataSource bool) map[string]*schema.Schema {
```

Comment:
Move the constant and the function to a new file `schema.go`.

Reply:
Addressed: [Move schema to separate file](https://github.com/cyralinc/terraform-provider-cyral/pull/479/commits/3abe2e4960d3bae3650e6be189b85a1a004faac5)